packages/demo-next/pages/nesting.tsx

```diff
@@ -186,6 +186,7 @@ const ROW = {
         <div style={{ display: 'flex', flexDirection: 'row' }}>
           <InlineBlocks
             name="items"
+            layout="columns"
```

Comment:
This feels backwards to me, I would expect it to be `layout: row` (singular), which matches the CSS of the parent div `flexDirection: row`.

Reply:
Ah, that's a good point. I'll make that change.

Reply:
I'm also going to change 'layout' to 'direction' to make it more clear.

Reply:
Changed.